soc/silabs/silabs_s0/Kconfig

```diff
@@ -5,3 +5,9 @@
 config SOC_FAMILY_SILABS_S0
 	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
 	select BUILD_OUTPUT_HEX
+
+if SOC_FAMILY_SILABS_S0
+
+rsource "*/Kconfig"
+
+endif # SOC_FAMILY_SILABS_S0
```

Comment:
For a future PR: because of Don't Repeat Yourself principle, I tend to avoid to repeat the condition after `endif` when it is obvious.